### PR TITLE
Remove unused and obsolete dependencies

### PR DESCRIPTION
**Pull request by henning-gerhardt**

### CODE REVIEW

File: Kitodo/pom.xml [hunk 1]
```diff
@@ -52,38 +52,6 @@
...
-            <artifactId>jcommon</artifactId>
-            <version>1.0.0</version>
-            <exclusions>
-                <exclusion>
-                    <groupId>junit</groupId>
-                    <artifactId>junit</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-        <dependency>
```
Comment (Beacze) [inline Kitodo/pom.xml]: This library is bring back to the project by UGH.

Comment (henning-gerhardt) [reply Kitodo/pom.xml]: Thank you. I will re-add this dependency as it is needed.

File: Kitodo/pom.xml [hunk 2]
```diff
@@ -619,11 +550,6 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
-            <groupId>org.primefaces.extensions</groupId>
-            <artifactId>resources-codemirror</artifactId>
```
[the remark this answers is not in the document]
Comment (solth) [reply Kitodo/pom.xml]: `resources-codemirror` is a specific Primefaces extension that requires the `primefaces-extension` itself, so both artifacts are required.

[the remark this answers is not in the document]
Comment (henning-gerhardt) [reply Kitodo/pom.xml]: Ah, ok. Both looks similar but `resources-codemirror` is extending  `primefaces-extensions` with functionality. I will re-add this dependecy including an additional comment.

**Pull request merged**